README.md

```diff
@@ -75,8 +78,8 @@ After bootstrap, you can use [Ansible] as usual to manage system services, insta
   become: yes
   tasks:
     - name: Start etcd
-      systemd: 
-        name: etcd.service 
+      systemd:
+        name: etcd.service
         state: started
 
     - name: Install docker-py
```

Comment:
@Adrian-Turjak, incorrect executable path in example.

Reply:
Done